docs/make.jl

```diff
@@ -0,0 +1,203 @@
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "Zygote"),
+                        path = "zygote",
+                        name = "Zygote",
+                        giturl = "https://github.com/FluxML/Zygote.jl.git",
+                    ),
+                ],
+            ),
+            MultiDocumenter.Column(
+                "Neural Network primitives",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "NNlib"),
+                        path = "nnlib",
+                        name = "NNlib",
+                        giturl = "https://github.com/FluxML/NNlib.jl.git",
+                    ),
+                ],
+            ),
+        ]
+    ),
+    MultiDocumenter.MegaDropdownNav(
+        "Training",
+        [
+            MultiDocumenter.Column(
+                "Data Wrangling",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "MLUtils"),
+                        path = "mlutils",
+                        name = "MLUtils",
+                        giturl = "https://github.com/JuliaML/MLUtils.jl.git",
+                    ),
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "OneHotArrays"),
+                        path = "onehotarrays",
+                        name = "OneHotArrays",
+                        giturl = "https://github.com/FluxML/OneHotArrays.jl.git",
+                    ),
+                ]
+            ),
+            MultiDocumenter.Column(
+                "Data Augmentation",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "DataAugmentation"),
+                        path = "dataaugmentation",
+                        name = "DataAugmentation",
+                        giturl = "https://github.com/FluxML/DataAugmentation.jl.git",
+                    ),
+                ],
+            ),
+            MultiDocumenter.Column(
+                "Datasets",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "MLDatasets"),
+                        path = "mldatasets",
+                        name = "MLDatasets",
+                        giturl = "https://github.com/JuliaML/MLDatasets.jl.git",
+                    ),
+                ],
+            ),
+            MultiDocumenter.Column(
+                "Schedulers",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "ParameterSchedulers"),
+                        path = "paramschedulers",
+                        name = "ParameterSchedulers",
+                        giturl = "https://github.com/FluxML/ParameterSchedulers.jl.git",
+                    ),
```

Comment:
ParameterSchedulers.jl uses Publish, which is still a little funky when loading with MultiDocumenter but integrates much better than Pollen

Reply:
We can definitely move this repo to Documenter.